opm/test_util/EclFilesComparator.cpp

```diff
@@ -108,15 +108,15 @@ void ECLFilesComparator::deviationsForCell(double val1, double val2, const std::
         std::cout << "(value1, value2) = (" << val1 << ", " << val2 << ")\n\n";
     }
     if (val1 < 0) {
-        if (abs(val1) > absTolerance) {
+        if (std::abs(val1) > absTolerance) {
```

Comment:
there is actually a system to the madness. sort of. abs() is a C function, for floats it is fabs().

however, in c++ there is std::abs which is a template. so if somebody has include <cmath> and flattened the std namespace, it resolves to that template. this is why it sometimes work as expected without you doing anything (but that's all an accident). you have three choices;

1) use fabs() - that always works but you are doing C not C++, this might bite in templates.
2) explicit qualification like here
3) using namespace std, or better using std:.abs;

i prefer 2) because there's no possibility for surprises that way (that somehow the wrong symbol get used on a stupid compiler or whatever).

the whole mess is basically a C compatibility issue. to add to the fun: visual studio always exposes the template so this is one of those watch-out-for points when i port visual studio code..